docs/scheduling-api/endpoints/appointments.md

```diff
@@ -17,6 +17,7 @@ Bearer Token `<token>`
 | Key        | Example value        | Required |
 |------------|----------------------| -------- |
 | sorCode    | 123ABC               | ✅       |
+| priority   | priority2            | ✅       |
```

Comment:
In the configuration example, we have the following:
 `"priority": "priority 2",` but in the example query above, we're showing priority set to `priority2`. 
I wonder if this would be confusing regarding how a priority should be set - with whitespace or no whitespace, etc.

Reply:
I've added spaces to the priority examples that didn't have them, which should make things consistent